contracts/utils/convex/ConvexStakingWrapper.sol

```diff
@@ -51,6 +49,7 @@ contract ConvexStakingWrapper is ERC20 {
 
     //rewards
     RewardType[] public rewards;
+    mapping(address => uint256) public registeredRewards;
```

Comment:
What's this for?

Reply:
Used to keep track of the rewards that are added. Makes the `addRewards` robust & prevent any duplication.